benchmark/buffers/buffer-base64-encode.js

```diff
@@ -1,16 +1,20 @@
 'use strict';
 var common = require('../common.js');
 
-var bench = common.createBenchmark(main, {});
+const bench = common.createBenchmark(main, {
+  N: [64 * 1024 * 1024],
+  n: [32]
+});
 
 function main(conf) {
-  var N = 64 * 1024 * 1024;
+  var n = conf.n;
+  var N = conf.N;
```

Comment:
Similarly for these, the values should be prefixed with `+` to ensure they are numbers.